src/sources/aws_kinesis_firehose/mod.rs

```diff
@@ -40,6 +40,8 @@ pub struct AwsKinesisFirehoseConfig {
     #[configurable(metadata(docs::examples = "A94A8FE5CCB19BA61C4C08"))]
     access_key: Option<SensitiveString>,
 
+    access_keys: Option<Vec<SensitiveString>>,
```

Comment:
You'll want to add the `configurable` macro here that you see on the above options as well as a doc comment as this comment will be used to drive the documentation.